models/Order.js

```diff
@@ -14,6 +14,7 @@ const SingleOrderItemSchema = mongoose.Schema({
 
 const OrderSchema = mongoose.Schema(
   {
+    // validations on min for the number fields?
```

Comment:
just a suggestion to include a `min` validation for the Number fields (for example, you probably don't want to allow saving `tax` with a negative value)